apps/guardian/src/pages/Actions/DecryptionActions.tsx

```diff
@@ -187,15 +187,19 @@ export function DecryptionActions() {
                     Waiting for peers to register their shares of the decryption.
                 </AwaitPeers>
             )}
+            {step === TallyStep.Incomplete && (
+                <h3>The election decryption was not completed by a sufficient amount of guardians</h3>
```

Comment:
```suggestion
                <h3>The decryption was not completed by a sufficient amount of guardians</h3>
```
or "election result decryption"